FLEDGE.md

```diff
@@ -109,7 +109,7 @@ The browser will remain in an interest group for only a limited amount of time.
 
 The `userBiddingSignals` is for storage of additional metadata that the owner can use during on-device bidding, and the `trustedBiddingSignals*` attributes provide another mechanism for making real-time data available for use at bidding time.
 
-The `biddingWasmHelperUrl` field is optional, and lets the bidder provide computationally-expensive subroutines in WebAssembly, rather than JavaScript, to be driven from the JavaScript function provided by `biddingLogicUrl`. If provided, it must point to a WebAssembly binary, delivered with a `application/wasm` mimetype. The corresponding `WebAssembly.Module` will be made available by the browser to the `scoreAd` function.
+The `biddingWasmHelperUrl` field is optional, and lets the bidder provide computationally-expensive subroutines in WebAssembly, rather than JavaScript, to be driven from the JavaScript function provided by `biddingLogicUrl`. If provided, it must point to a WebAssembly binary, delivered with a `application/wasm` mimetype. The corresponding `WebAssembly.Module` will be made available by the browser to the `generateBid` function.
```

Comment:
I'm not entirely convinced that `browserSignals` is the right place to attach the WASM as the WASM isn't exactly a signal coming from the browser like the other `browserSignals` members, almost seems better suited to an additional `generateBid` argument, but at the same time it seems awkward or wasteful to make every `generateBid() `method change their prototype to include an extra WASM parameter if some/many of them aren't using WASM.  Anyhow, I don't have a better solution in mind.

My comment here is: should we add a blurb here mentioning how it's made available to `generateBid`?  e.g. "...as a `wasmHelper` member of `browserSignals`"

Reply:
> I'm not entirely convinced that `browserSignals` is the right place to attach the WASM as the WASM isn't exactly a signal coming from the browser like the other `browserSignals` members

Me neither, which is why I wanted feedback.

>almost seems better suited to an additional `generateBid` argument, but at the same time it seems awkward or wasteful to make every `generateBid() `method change their prototype to include an extra WASM parameter if some/many of them aren't using WASM. 

If we add it to the end, they don't have to change it their signature --- extra arguments are ignored. Of course, there is only so often you can pull that trick.

> Anyhow, I don't have a better solution in mind.

A global is another option, but it feels awkward. 

> 
> My comment here is: should we add a blurb here mentioning how it's made available to `generateBid`? e.g. "...as a `wasmHelper` member of `browserSignals`"

Seems redundant?

Reply:
I wonder if this is an indication that these functions should be taking an options bag, and not positional arguments?